iphone/Maps/Categories/MWMCategory+PlacesCountTitle.swift

```diff
@@ -2,13 +2,18 @@ import Foundation
 
 extension BookmarkGroup {
   @objc func placesCountTitle() -> String {
-    if (bookmarksCount > 0 && trackCount > 0) ||
-      (bookmarksCount == 0 && trackCount == 0) {
-       return String(format: L("objects"), bookmarksCount + trackCount)
-    } else if (bookmarksCount > 0) {
-      return String(format: L("bookmarks_places"), bookmarksCount)
-    } else {
-       return String(format: L("tracks"), trackCount)
+    let bookmarks = String(format: L("bookmarks_places"), bookmarksCount)
+    let tracks = String(format: L("tracks"), trackCount)
+
+    if (bookmarksCount == 0 && trackCount == 0) || (bookmarksCount > 0 && trackCount > 0) {
+      return "\(bookmarks), \(tracks)"
+    }
+
+    if (bookmarksCount > 0) {
+      return bookmarks
     }
+
+    return tracks
+
```

Comment:
```suggestion
    if (bookmarksCount == 0 && trackCount == 0 || bookmarksCount > 0 && trackCount > 0) {
      return String(format: L("comma_separated_pair"),
                    String(format: L("bookmarks_places"), bookmarksCount),
                    String(format: L("tracks"), trackCount))
    }

    if (bookmarksCount > 0) {
      return String(format: L("bookmarks_places"), bookmarksCount)
    }

    return String(format: L("tracks"), trackCount)
```

@kirylkaveryn for some reason, singular forms are not printed with this code on iOS 15 simulator. Do you have any ideas why it doesn't work?

Reply:
Hmm it seems like the plurals have only the `other` field and miss the `one` for EN.

![image](https://github.com/organicmaps/organicmaps/assets/79797627/1416b41f-8bfb-42da-b3d9-e628ddd409b5)

Reply:
@kirylkaveryn thanks for the hint. Looks like our Twine tool does not generate plurals from a generic language (en) into sublanguage (en-GB):
```
diff iphone/Maps/LocalizedStrings/en.lproj/Localizable.stringsdict iphone/Maps/LocalizedStrings/en-GB.lproj/Localizable.stringsdict
6c6
<  * Language: en -->
---
>  * Language: en-GB -->
22,23d21
<       <key>one</key>
<       <string>%d bookmark</string>
39,40d36
<       <key>one</key>
<       <string>%d file was found. You can see it after conversion.</string>
56,57d51
<       <key>one</key>
<       <string>%d track</string>
```